tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCheckCLDR.java

```diff
@@ -467,13 +467,26 @@ public void TestFullErrors() {
         checkLocale(test, localeID, "?", null);
     }
 
-    public void TestAllLocales() {
+    /** adjust the logging level of checks */
+    public java.util.logging.Level pushCheckLevel() {
         java.util.logging.Level oldLevel = null;
         if (logKnownIssue(
                 "CLDR-17320",
-                "turning off CheckCLDR logging to avoid 2,000 log messages, please fix internal stack traces")) {
+                "turning off CheckCLDR logging to avoid thousands of log messages, please fix internal stack traces")) {
             oldLevel = CheckCLDR.setLoggerLevel(java.util.logging.Level.OFF);
         }
+        return oldLevel;
+    }
+
+    /** undo the effect of a popCheckLevel */
```

Comment:
Shouldn't this be "undo the effect of a pushCheckLevel" ?

Reply:
fixed thanks